src/js/formapp.js

```diff
@@ -58,21 +58,24 @@ function getContext(contextScripts) {
 
 let prevSubmission;
 
+function updateSubmision() {
+  router.request('update:storedSubmission', prevSubmission);
+}
+
+const updateSubmisionDebounce = debounce(updateSubmision, /* istanbul ignore next */ _TEST_ ? 100 : 2000);
+
 const onChange = function(form, changeReducers) {
   const data = getChangeReducers(form, changeReducers, structuredClone(form.submission.data), prevSubmission);
 
   form.data = data;
   form.setSubmission({ data }, { fromChangeReducers: true, fromSubmission: false });
 
   prevSubmission = structuredClone(form.submission.data);
+  updateSubmisionDebounce();
```

Comment:
Missing an `s` in `submission` in a few places 👀 , `updateSubmision` vs. `updateSubmission`

Reply:
Looks like that was there from before these changes...

Reply:
`s's are expenive

Reply:
😆

Reply:
> `s's are expenive

See, that's why I had no problem leaving those off. 😆